provider/k8s/process.go

```diff
@@ -150,6 +159,21 @@ func (p *Provider) ProcessList(app string, opts structs.ProcessListOptions) (str
 		pss = append(pss, *ps)
 	}
 
+	// ignore error because this api only exists if metrics crds are installed in the k8s
+	ms, err := p.MetricsClient.MetricsV1beta1().PodMetricses(p.AppNamespace(app)).List(am.ListOptions{LabelSelector: strings.Join(filters, ",")})
+	if err == nil {
```

Comment:
I think we can return the error here. It could be useful to notify the client about problems. We include metrics server by default in the rack instalation so if it's missing we'd know.

Reply:
addressed it in the next commit